YourFirstPR.md

```diff
@@ -42,13 +42,31 @@ You can also run those steps independently or on a more fine grained way.
 
 Make sure to run the automated tests using `bundle exec` to ensure you’re running the correct version of `rspec` and `rubocop`
 
-First, navigate into the root of the _fastlane_ project and run unit tests using
+#### All unit tests
+
+First, navigate into the root of the _fastlane_ project and run all unit tests using
 
 ```
 bundle exec rspec
 ```
 
-If you want to run tests only for one tool, use `bundle exec rspec [tool_name]`
+#### Unit tests for one specific tool
+
+If you want to run tests only for one tool, use
+
+```
+bundle exec rspec [tool_name]
+```
+
+#### Unit tests in one specific test file
+
+If you know exactly which `_spec.rc` file you want to run, use
+
+```
+bundle exec rspec .\fastlane\spec\fastlane_require_spec.rb
```

Comment:
Hmmm, Windows' back slashes, right?

Reply:
In Windows both work, I think this happened via auto completion.

Reply:
No, on macOS and Linux, the separator is `/`, so I think it completely removes `\`:

> LoadError: cannot load such file -- /Users/iulian/Development/GitHub/fastlane/fastlane/.fastlanespecfastlane_require_spec.rb